### PR TITLE
Update to 1.15.1 and fix mixin crash

### PR DESCRIPTION
**Pull request by Siphalor**
That's all :)

### CODE REVIEW

File: src/main/java/dynamicfps/mixin/GameRendererMixin.java [hunk 1]
```diff
@@ -23,7 +22,7 @@
 	 Implements the mod's big feature.
 	 */
 	@Inject(at = @At("HEAD"), method = "render", cancellable = true)
-	private void onRender(CallbackInfo callbackInfo) {
+	private void onRender(float tickDelta, long limitTime, boolean tick, CallbackInfo callbackInfo) {
```
Comment (juliand665) [inline src/main/java/dynamicfps/mixin/GameRendererMixin.java]: Interesting, so mixin requires you to specify the arguments now?

Comment (Siphalor) [reply src/main/java/dynamicfps/mixin/GameRendererMixin.java]: Seems like it - without arguments the mixin couldn’t be applied 🤷

Comment (juliand665) [reply src/main/java/dynamicfps/mixin/GameRendererMixin.java]: Huh, interesting. I'll have to check for myself, but that's good to know!

**Comment by juliand665** [comment]
Thanks a lot for this! I just ran the updated version and it worked great (after invalidating intellij's cache and restarting, lol).

However, it doesn't seem like the additional mixin arguments are necessary after all—it worked just fine for me without them, no errors/warnings in the console or anything.

Can you test it with [this build](https://github.com/juliand665/Dynamic-FPS/files/3998857/dynamic-fps-1.0.4.jar.zip) and report back if it breaks?

**Comment by juliand665** [comment]
I went ahead and uploaded it now—will keep this mixin stuff in the back of my mind if there are crashes.

**Pull request merged**